cli/create.go

```diff
@@ -61,20 +61,6 @@ func create() *cobra.Command {
 				}
 			}
 
-			tz, err := time.LoadLocation(tzName)
-			if err != nil {
-				return xerrors.Errorf("Invalid workspace autostart timezone: %w", err)
-			}
```

Comment:
**Question**: Correct me if I'm wrong, I feel like this validation is just gone now from the code (IE is not below or in `validSchedule`. Just wanted to gut-check that and ask why it's not needed since I'm less familiar over here.

Reply:
It will still be validated on the backend, but good catch - when creating a workspace, I'd like to do as much of the validation up-front as possible in the CLI before the dry-run gets sent off.